spec/spec_helper.rb

```diff
@@ -0,0 +1,109 @@
+require 'omniauth'
+
+OmniAuth.config.test_mode = true
+
+omniauth_hash = OmniAuth::AuthHash.new({
+  'provider': 'auth_provider',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.